grantnav/frontend/templates/components/search-summary-box.html

```diff
@@ -141,7 +141,7 @@
             <span class="summary-footer--item__value">
               {{currency|currency_symbol}}{{results.aggregations.earliest_grant.hits.hits.0|get:"_source"|get:"amountAwarded"|human_format}}
               from
-              <a href="grant/{{results.aggregations.earliest_grant.hits.hits.0|get:"_source"|get:"fundingOrganization"|first|get:"id"}}">
+              <a href="grant/{{results.aggregations.earliest_grant.hits.hits.0|get:"_source"|get:"id"}}">
```

Comment:
Can we use django `reverse` in templates?

Reply:
That would be better yeah. Gonna leave it as out of scope for this time though.